__tests__/authButton.test.jsx

```diff
@@ -0,0 +1,30 @@
+import { screen, render } from "@testing-library/react";
+import AuthButton from "../components/authButton";
+import "@testing-library/jest-dom";
+import { useSession } from "next-auth/react";
+
+jest.mock('next-auth/react', () => {
+    const originalModule = jest.requireActual('next-auth/react');
+    return {
+        __esModule: true,
+        ...originalModule,
+        useSession: jest.fn() 
+    };
+});
+
+describe("Auth Button", () => {
+    it('shows Sign out when session exists', async () => {
```

Comment:
```suggestion
    it('shows Sign out when session exists', () => {
```
It's a synchronous function.